src/Command/Shared/ConfirmationTrait.php

```diff
@@ -18,7 +18,7 @@
      *
      * @return bool
      */
-    public function confirmGeneration()
+    public function confirmOperation()
```

Comment:
@LOBsTerr could we add this method to avoid breaking contrib modules using it
```
    /**
     * @deprecated
     */
    public function confirmGeneration()
    {
        return  $$this->confirmOperation();
    }
```

Reply:
Yes, it is true. I will do it

Reply:
Awesome thanks

Reply:
Done